dask_ml/model_selection/_incremental.py

```diff
@@ -25,6 +28,7 @@
 from ._split import train_test_split
 
 Results = namedtuple("Results", ["info", "models", "history", "best"])
+logger = logging.getLogger(__name__)
```

Comment:
I'm not sure, but I think this should be

```suggestion
logger = logging.getLogger("dask_ml.model_selection")
```

We want users to configure `'dask_ml.model_selection'`, not 'dask_ml.model_selection._incremental'`

alternatively, in `dask_ml/model_selection/__init__.py` you can do


```python
logger = logging.getLogger(__name__)
```

and try `from . import logger`, but that may cause a circular import.